ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -165,13 +170,19 @@ open class ShimmerView: UIView {
 
 	/// Update the frame of each layer covering views in the containerView
 	func updateViewCoverLayers() {
-		guard let containerView = containerView else {
-			return
-		}
+		let viewToCover = containerView ?? self
 
 		viewCoverLayers.forEach { $0.removeFromSuperlayer() }
 
-		let subviews = Set(containerView.subviews).subtracting(Set(excludedViews))
+		let subviews: Set<UIView> = {
+			if shimmersLeafViews {
+				var leaves = [UIView]()
+				searchLeaves(in: viewToCover, output: &leaves)
+				return Set(leaves).subtracting(Set(excludedViews))
+			} else {
+				return Set(viewToCover.subviews).subtracting(Set(excludedViews))
+			}
+		}()
```

Comment:
yay for a let with a boolean on the inside! This is instead of writing a super weird looking ternary statement, or making subviews be a var 😄